components/Carousel/Carousel.js

```diff
@@ -17,3 +20,58 @@
     <div class="right-button"> > </div>
   </div>
 */
+
+const carouselContainer = document.querySelector('.carousel-container');
+// const create = (el) => document.createElement(el);
+
+function createCarousel() {
```

Comment:
How did you go about creating your carousel? Did you find an article or video? Or just plan it out?